src/rnp/fficli.cpp

```diff
@@ -668,6 +668,29 @@ cli_key_usage_str(rnp_key_handle_t key, char *buf)
     return orig;
 }
 
+std::string
+cli_rnp_escape_string(const char *src)
```

Comment:
As we are (rather slowly) moving to the C++, new code (especially on CLI level) should be as much as possible in C++, i.e. here please use std::string as parameter. Not sure about the escape_map.